spec.yaml

```diff
@@ -355,6 +355,47 @@ paths:
               type: string
           schema:
             $ref: "#/definitions/Error"
+  /invites/forward:
+    post:
+      summary: Forward an invitation to start sharing
+      description: Forwards an invitation received by an out-of-band channel, email for example.
```

Comment:
Maybe you mean "**through** an out-of-band channel"?

Reply:
That's a nice improvement, I'll update.